### PR TITLE
buzzinga-test

### PR DESCRIPTION
**Pull request by zainabdnaya**

### CODE REVIEW

**Review by Hamzaelkhatri — APPROVED** [review]

File: solutions/buzzinga.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+import "fmt"
+
+func BuzZinga(n int32) {
+	var i int32 = 0
+	if n == 0 {
+		fmt.Println("Buzzinga")
+	}
+	if n < 0 {
+		fmt.Print("*")
```
Comment (MSilva95) [inline solutions/buzzinga.go:11]: missing newline for negatives

**Review by MSilva95 — CHANGES_REQUESTED** [review]

File: tests/buzzinga_test/main.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+import (
+	student "student"
+
+	"github.com/01-edu/go-tests/lib/challenge"
+	"github.com/01-edu/go-tests/solutions"
+)
+
+func main() {
+	challenge.Function("BuzZinga", student.BuzZinga, solutions.BuzZinga, 20)
```
[the remark this answers is not in the document]
Comment (zainabdnaya) [reply tests/buzzinga_test/main.go:11]: corrected

File: solutions/buzzinga.go (hunk 1 above)
Comment (zainabdnaya) [reply solutions/buzzinga.go]: corrected

File: tests/buzzinga_test/main.go [hunk 3]
```diff
@@ -0,0 +1,25 @@
+package main
```
Comment (MSilva95) [inline tests/buzzinga_test/main.go:1]: Try to run the test, its not working...
You should define the int32 and use an array to keep it clean...
something like this
```
func main() {
	args := []int32{20, 15, -125, 10, 5, 4, -9, 9, 24, 0, -85, 06, 11}

	for _, arg := range args {
		challenge.Function("BuzZinga", student.BuzZinga, solutions.BuzZinga, arg)
	}
}
```

**Review by MSilva95 — CHANGES_REQUESTED** [review]

**Review by MSilva95 — APPROVED** [review]

**Pull request merged**